lunes_cms/help/templates/public_upload.html

```diff
@@ -65,28 +65,46 @@
         3: "das",
         4: "die (Plural)"
       }
+      var buffer_array = []
```

Comment:
```suggestion
      let buffer_array = []
```
Please use [let instead of var](https://stackoverflow.com/a/21907082) wherever possible.